static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -268,4 +268,9 @@
         }
     }
 
+    .media-primary & {
+        @include mq($from: mobile, $until: mobileLandscape) {
+            margin-top: -1rem;
```

Comment:
Yep I looked at removing the margin initially, however the margin is on figure.element which I thought was quite a high-level element/class to make this change to. So I then looked into updating the Scala template to see if I could add an additional class to the figure.element if it wrapped a youtube-atom that is used as main media, so I could target it specifically in the CSS. However the figure.element isn't in a template it's actually embedded in the markup from CAPI, so I'd need to parse that and update it there. This seemed easier!